english/eating-walls/find-a-toilet/door/bear/too-many-marshmallows/get-eaten.md

```diff
@@ -0,0 +1 @@
+The bear turns round, growls at you, and then in a fit of rage grasps you in a bear hug and starts eating you from the head down.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.